src/dfu.cpp

```diff
@@ -8,8 +8,10 @@
 #define DFU_STATE_HUMAN 200
 #define DFU_STATE_HELP 201
 
+
 uint8_t dfu_state = DFU_STATE_MAIN;
 uint8_t dfu_human_mode = false;
+uint8_t dfu_update_flag = false;
```

Comment:
Consider renaming it "dfu_node_activated" or something to signify the radio is in active state (i.e. needs to be `update()`'d)